packages/vue/src/components/templates/SfOrderSummary/SfOrderSummary.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div class="sf-order-summary">
+    <slot name="heading">
+      <SfHeading
+        :title="orderTitle"
+        :level="orderTitleLevel"
+        class="
+          sf-heading--left sf-heading--no-underline
+          sf-order-summary__heading
+        "
+      />
+    </slot>
+    <div class="highlighted highlighted--total">
+      <slot name="summary">
+        <SfProperty
+          name="Products"
```

Comment:
properties names could be changed by props

Reply:
Thanks good point :)